src/prog_models/models/uav_model/vehicles/AircraftModels.py

```diff
@@ -0,0 +1,268 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration.  All Rights Reserved.
+
+"""
+Aircraft Models - originally developed by Matteo Corbetta (matteo.corbetta@nasa.gov) for SWS project
+"""
+import numpy as np
+
+import sys
+import os
+sys.path.insert(1, os.path.join(os.path.dirname(__file__), '../utilities/'))
+
+from prog_models.models.uav_model.vehicles.control import controllers
+from prog_models.models.uav_model.vehicles.aero import aerodynamics as aero
+import prog_models.models.uav_model.utilities.utils as utils
+import prog_models.models.uav_model.vehicles.vehicles as vehicles
+from prog_models.exceptions import ProgModelException
+
+
+# Initialize rotorcraft
+# =====================
+def build_model(**kwargs):
+
+    params = dict(name='rotorcraft-1', model='djis1000',
+                  init_state_vector=None, dt=None,
+                  payload=0.0, Q=None, R=None, qi=None, i_lag=None,
+                  steadystate_input=None)
+    params.update(kwargs)
+    
+    # Default control parameters (if not provided)
+    if params['Q'] is None:
+        params['Q'] = np.diag([1000, 1000, 25000,  # x, y, z
+                               100.0, 100.0, 100.0,  # phi, theta, psi,
+                               1000, 1000, 5000,  # vx, vy, vz,
+                               1000, 1000, 1000])  # p, q, r
+    if params['R'] is None:         params['R']     = np.diag([500, 4000, 4000, 4000])   # T, Mu, Mv, Mw
+    if params['qi'] is None:        params['qi']    = np.array([100, 100, 1000])  # Integral error weights in position x, y, z
+    if params['i_lag'] is None:     params['i_lag'] = 100                        # integral lag: how far back to use (in data points, so 1 point = 1 dt) for integral error
+
```

Comment:
here I cleaned up some controller parameter's definition which have been there for a while (my fault). The trajectories now look a little cleaner as well.